lib/browser/api/app.js

```diff
@@ -77,6 +77,24 @@ app.allowNTLMCredentialsForAllDomains = function (allow) {
   }
 }
 
+const makeSingleInstance = app.makeSingleInstance
+app.makeSingleInstance = function (callback) {
+  return makeSingleInstance.call(app, function (argv, ...args) {
+    if (process.platform === 'win32') {
+      // Delay the checking so the callback is run as quickly as we would expect
+      setImmediate(() => {
+        argv.forEach((arg) => {
+          const protocolMatch = arg.match(/^([\s\S]+):\/\/[\s\S]+/)
```

Comment:
@kof I have that locally.  I mentioned in our other exchange as well that this wasn't finished.  I didn't put WIP in the PR title though so my bad.